packages/in_app_purchase/in_app_purchase_storekit/shared/Classes/FIAPaymentQueueHandler.h

```diff
@@ -125,7 +125,7 @@ typedef void (^UpdatedDownloads)(NSArray<SKDownload *> *downloads);
 // - You have increased the price of the subscription in App Store Connect.
 // - The subscriber has not yet responded to a price consent query.
 // Otherwise the method has no effect.
-- (void)showPriceConsentIfNeeded API_AVAILABLE(ios(13.4));
+- (void)showPriceConsentIfNeeded API_AVAILABLE(ios(13.4))API_UNAVAILABLE(tvos, macos, watchos);
```

Comment:
Nit: missing space before the new macro.

Reply:
The formatter complains about the space being there and running [the auto-formatter](https://github.com/flutter/plugins/blob/main/script/tool/README.md#format-code) results in the space being removed again.

Reply:
Ah, must be a minor autoformatter bug. Sorry for the noise!

Reply:
No worries